ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -128,6 +128,9 @@ class TransactionSummaryScreen extends React.Component<Props> {
       error
         .filter(_ => _ === "PAYMENT_DUPLICATED")
         .map(_ => this.props.onDuplicatedPayment());
+      if (error.isSome()) {
```

Comment:
I think we don't have to show the error screen if the type of the error is PAYMENT_DUPLICATED.

cc @cloudify

Reply:
@laurapugliese Can you check this.

Reply:
yes we have to, the user always needs to know why he can't pay - he may not know (i.e. somebody else in the family paid already) or he may forgot

Reply:
@fpersico @cloudify so is it correct how I implemented it?

Reply:
@fpersico if the error is PAYMENT_DUPLICATED before this function was called:

```
onDuplicatedPayment: () =>
      dispatch(
        paymentCompletedSuccess({
          rptId: props.navigation.getParam("rptId"),
          kind: "DUPLICATED"
        })
      )
    }
```
Do you think it compromises something to remove this action?